### DISCUSSION TITLE
How to set taskbar shortcut to "current"?

### DISCUSSION THREAD

**Discussion by vncastanheira** [ScoopInstaller/Scoop#6021] [Help and Questions]
Something that have been bothering a while is that if you set a shortcut on the taskbar it creates a path linked directly to the executable, which means that if you update the app it's gonna open the old version, This is specially a problem for browsers that updates frequently.

Is there a way to set a taskbar shortcut to always point to the "current" version or is this a limitation from Windows?

I'm using Windows 11.

**Comment by HUMORCE** [comment]
> Is there a way to set a taskbar shortcut to always point to the "current" version or is this a limitation from Windows?

Drag and drop shortcut from shortcut dir(Scoop Apps) to Taskbar.

> or is this a limitation from Windows?

In some cases, the icons may not activate when open with it.

Example:

Chromium / Everything / 7-Zip are the testcases

![1](https://github.com/user-attachments/assets/5356037e-f22f-4e41-be5d-bfe761a74e13)
![1_1](https://github.com/user-attachments/assets/695fb885-34ce-4e3c-9043-fe8195db620d)

a. When start them:
- Chromium: not working
- Everything: fine
- 7-zip: fine

![2](https://github.com/user-attachments/assets/bd3efb3a-1638-4dbb-bfb4-77d3126ccbdf)

b. After a system restart:

- Chromium: not working
- Everything: not working
- 7-zip: fine

![3](https://github.com/user-attachments/assets/cd429f06-e959-4006-a1a2-d9089f797973)

In section a, I do a full-exit for the Everything App first.
In section b, Everything App started on system startup and it already running in background.
